.github/workflows/publish_layer.yml

```diff
@@ -0,0 +1,25 @@
+name: Publish lambda layer to SAR
+
+on:
+  release:
+    types: [published]
+env:
+  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  AWS_DEFAULT_REGION: eu-west-1
+  AWS_DEFAULT_OUTPUT: json
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set release notes tag
+      run: |
+        export RELEASE_TAG_VERSION=${{ github.event.release.tag_name }}
+        echo ::set-env name=RELEASE_TAG_VERSION::${RELEASE_TAG_VERSION:1}
+    - name: write version to ssm
+      run: |
+        aws ssm put-parameter --name "powertools-python-release-version" --value $RELEASE_TAG_VERSION --overwrite
+    - name: trigger lamber layer publish pipeline
+      run: |
+        aws codepipeline start-pipeline-execution --name ${{ secrets.CODEPIPELINE_NAME }}
```

Comment:
Two questions, if I may: 

* When the pipeline starts, where does it pull the latest code from? Using the release tag version?
* Should this run after publishing to PyPi? If so, then this can be merged into the `publish` workflow with a new task that depends on tests, linting, etc.

Reply:
the code is pulled from pypi repo, no connection to the GitHub repo, so we can merge it with `publish`, good catch!